### DISCUSSION TITLE
Cannot install on Debian Sid

### DISCUSSION THREAD

**Discussion by stangosnell** [roddhjav/pass-import#224] [General]
I cannot install pass-import on Debian Sid because of dependency problems.  Neither python3-pretty-yaml nor python3-zxcvbn is installable.  Any chance of an update to fix this?

**Comment by roddhjav** [comment]
It is not python3-pretty-yaml but [python3-yaml](https://packages.debian.org/sid/python3-yaml) and it is available on Debian Sid.

It seems they are now going to use https://packages.debian.org/sid/python3-python-zxcvbn-rs-py instead of python3-zxcvbn, so you are in luck, as it is faster than the full python version.
